al-cwe-collector.json

```diff
@@ -1,7 +1,6 @@
 {
     "Runtime": {
         "path": "Runtime",
-        "value": "nodejs8.10"
+        "value": "nodejs10.16.3"
```

Comment:
please change it to `nodejs10.x`
https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html